src/storage/mutate/UpdateEdgeProcessor.cpp

```diff
@@ -51,8 +51,16 @@ void UpdateEdgeProcessor::doProcess(const cpp2::UpdateEdgeRequest& req) {
     onFinished();
     return;
   }
-
-  planContext_ = std::make_unique<PlanContext>(env_, spaceId_, spaceVidLen_, isIntId_);
+  auto session_id =
+      req.common_ref().has_value() && req.common_ref().value().session_id_ref().has_value()
+          ? *req.get_common()->get_session_id()
+          : 0;
+
+  auto plan_id = req.common_ref().has_value() && req.common_ref().value().plan_id_ref().has_value()
+                     ? *req.get_common()->get_plan_id()
+                     : 0;
+  planContext_ =
+      std::make_unique<PlanContext>(env_, spaceId_, session_id, plan_id, spaceVidLen_, isIntId_);
```

Comment:
Why not provide a PlanContext constructor for `req.common_ref()`? In the constructor, you can check whether plan id and session id is valid? and The caller is also easier to use, such as:

```cpp
planContext_ =
      std::make_unique<PlanContext>(env_, spaceId_, spaceVidLen_, isIntId_, req.common_ref());


PlanContext(StorageEnv* env,
              GraphSpaceID spaceId,
              size_t vIdLen,
              bool isIntId,
              cpp2::ReqCommon req)
      : env_(env),
        spaceId_(spaceId),
        vIdLen_(vIdLen),
        isIntId_(isIntId),
        sessionId_(0),
        planId_(0) {
  if (req.has_value()) {
    // handle session id and plan id
  }
}
```